staging/src/kubevirt.io/api/instancetype/v1beta1/types.go

```diff
@@ -297,6 +297,9 @@ const (
 
 	// Prefer vCPUs to be exposed as threads to the guest
 	PreferThreads PreferredCPUTopology = "preferThreads"
+
+	// Prefer vCPUs to be spread between cores and sockets
```

Comment:
`spread evenly between cores and sockets with any remaining vCPUs being presented as cores`